packages/user-mgnt/src/features/createUser/service.ts

```diff
@@ -123,30 +121,11 @@ export const createFhirPractitionerRole = async (
           ]
         }
       ],
-      location: (user.catchmentAreaIds || []).map((id) => ({
-        reference: `Location/${id}`
-      }))
+      location: [{ reference: `Location/${user.primaryOfficeId}` }]
     }
```

Comment:
This change is potentially breaking.. we need to look into the implications of this

Reply:
We also need a migration for this